server/src/main/java/com/objectcomputing/checkins/services/agenda_item/AgendaItemServicesImpl.java

```diff
@@ -65,15 +67,8 @@ public AgendaItem update(AgendaItem agendaItem) {
     }
 
     public Set<AgendaItem> findByFields(UUID checkinid, UUID createdbyid) {
```

Comment:
Not caused by this story specifically, but it would be worth putting a nullable annotation on these.

Reply:
updated, thanks